src/main/java/com/swervedrivespecialties/exampleswerve/commands/ToggleMinSpeedScale.java

```diff
@@ -0,0 +1,44 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package com.swervedrivespecialties.exampleswerve.commands;
+
+import com.swervedrivespecialties.exampleswerve.subsystems.DrivetrainSubsystem;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class ToggleMinSpeedScale extends CommandBase {
```

Comment:
I think I mentioned this before, but you should make this extends InstantCommand. You can create one from your VSCode menu, and it will greatly improve readability.

Reply:
fixed